app/components/shared/progress_card_component.rb

```diff
@@ -24,13 +24,18 @@ def step_number_for(step)
     def progress_card_step_component(step:, label:, step_at: nil)
```

Comment:
@mjgiarlo I think there may be some refactoring that can be done here, and possibly around the SubmissionPresenter to clean this component up a bit, but thought it would make sense to get this in and follow up with refactoring in case it becomes hairy.